FIRO_TSEnsembles/src/main/java/hec/stats/InlineStats.java

```diff
@@ -6,6 +6,7 @@ public class InlineStats {
     private int _sampleSize;
     private float _mean;
     private float _sampleVariance;
+    private float _standardDeviation;
```

Comment:
don't store standard deviation, it can be retrieved based on sample variance when requested.

Reply:
updated based on recommendation